react-function-components/src/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const element = <CustomButton />;
+const container = document.querySelector('#root');
+
+const root = ReactDOM.createRoot(container);
+
+root.render(element);
```

Comment:
This is okay, but you can skip creating the `element` `const` and directly pass the component to `render`. For example:
```JSX
root.render(<CustomButton />);
```